src/components/pathway-progress/SubscriptionStatusCard.jsx

```diff
@@ -0,0 +1,42 @@
+import React, { useContext } from 'react';
+import { Card, Badge } from '@edx/paragon';
+
+import { UserSubsidyContext } from '../enterprise-user-subsidy';
+import { LICENSE_STATUS } from '../enterprise-user-subsidy/data/constants';
+import { SubsidyRequestsContext, SUBSIDY_TYPE } from '../enterprise-subsidy-requests';
+
+function SubscriptionStatusCard() {
+  const {
+    subscriptionPlan,
+    subscriptionLicense: userSubscriptionLicense,
+  } = useContext(UserSubsidyContext);
+  const expirationDate = subscriptionPlan?.expirationDate;
+
+  const {
+    requestsBySubsidyType,
+  } = useContext(SubsidyRequestsContext);
+
+  const licenseRequests = requestsBySubsidyType[SUBSIDY_TYPE.LICENSE];
+  const hasActiveLicenseOrLicenseRequest = (subscriptionPlan
+    && userSubscriptionLicense?.status === LICENSE_STATUS.ACTIVATED) || licenseRequests.length > 0;
+
+  return (
+    <div className="row-cols-3 subscription-status-card">
+      <Card className="w-40">
+        <Card.Section
+          className="d-flex flex-column align-items-left justify-content-between"
+        >
+          <div className="h4 mb-0">
+            <span>Subscription Status</span>&nbsp; &nbsp;
```

Comment:
nit: This text should be made a constant

Reply:
Can you explain why? I think hardcoded text here is better.